components/bingx/actions/account-interface/bingx-get-balance.mjs

```diff
@@ -0,0 +1,29 @@
+import bingx from "../../bingx.app.mjs";
+
+export default {
+  name: "BingX Account GetBalance",
+  version: "0.0.1",
+  key: "bingx-account-get-balance",
+  description: "Get Perpetual Swap Account Asset Information [reference](https://bingx-api.github.io/docs/swap/account-api.html#_1-get-perpetual-swap-account-asset-information).",
+  props: {
+    bingx,
+    currency: {
```

Comment:
Could you please move this variable to the `app`?

Reply:
@vellames  - same as symbol. This also need to stay in the action. At present, I see USDT is the only currency. But, when multiple currencies are added, users should be able to call get-balance with different currencies.
Adding to app may just limit this.